test/rsht_test.c

```diff
@@ -14,13 +14,21 @@ bool lookerupper(rsht_entry *ent, void *userdata) {
   return true;
 }
 
-
 bool destructor(rsht_entry *ent, void *userdata) {
   free(ent->key);
   free(ent->val);
   return true;
 }
 
+// this will return false on the 5th iteration
+bool failer(rsht_entry *ent, void *userdata)
+{
+  static size_t fail_counter = 5;
+  if (--fail_counter == 0)
```

Comment:
This inline decrement is too strong for my tiny brain. Could you please put it on a separate line?

Reply:
Aye aye, captain.